frontends/web/src/routes/account/info/buyReceiveCTA.tsx

```diff
@@ -48,14 +55,21 @@ export const BuyReceiveCTA = ({ code, unit, balanceList }: TBuyReceiveCTAProps)
       <h3 className="subTitle">{t('accountInfo.buyCTA.information.start')}</h3>
       <div className={styles.container}>
         {balanceList && <Button primary onClick={onReceiveCTA}>{formattedUnit ? t('receive.title', { accountName: formattedUnit }) : t('receive.title', { accountName: t('buy.info.crypto') })}</Button>}
-        <Button primary onClick={onBuyCTA}>{formattedUnit ? t('accountInfo.buyCTA.buy', { unit: formattedUnit }) : t('accountInfo.buyCTA.buyCrypto')}</Button>
+        {exchangeBuySupported && <Button primary onClick={onBuyCTA}>{formattedUnit ? t('accountInfo.buyCTA.buy', { unit: formattedUnit }) : t('accountInfo.buyCTA.buyCrypto')}</Button>}
       </div>
     </div>);
 };
 
 
-export const AddBuyReceiveOnEmptyBalances = ({ balances }: {balances?: Balances}) => {
-  if (balances === undefined) {
+export const AddBuyReceiveOnEmptyBalances = ({ balances, accounts }: TAddBuyReceiveOnEmpyBalancesProps) => {
+
+  const [supportedAccounts, setSupportedAccounts] = useState<IAccount[]>();
+
+  useEffect(() => {
+    getExchangeSupportedAccounts(accounts).then(setSupportedAccounts);
+  }, [accounts]);
+
```

Comment:
This API call is used get all supported accounts. If there's more than 1 account that's supported, "Buy" button will be shown. If there's no supported accounts, it won't.